src/components/Breadcrumb/README.md

```diff
@@ -0,0 +1,12 @@
+# `<Breadcrumb />`
+
+DESCRIPTION_HERE
```

Comment:
Might as well outline the available props here so that it doesn't look like an incomplete component.